integrationtests/self/key_update_test.go

```diff
@@ -89,6 +89,7 @@ func (t *connTracer) DroppedKey(logging.KeyPhase)
 func (t *connTracer) SetLossTimer(logging.TimerType, logging.EncryptionLevel, time.Time) {}
 func (t *connTracer) LossTimerExpired(logging.TimerType, logging.EncryptionLevel)        {}
 func (t *connTracer) LossTimerCanceled()                                                 {}
+func (t *connTracer) Log(string, string)                                                 {}
```

Comment:
Maybe give it a more descriptive name if it's meant as "hack"? `Debug`?

Reply:
Good point. Done.